.ci/jenkins/config/main.yaml

```diff
@@ -4,6 +4,9 @@ ecosystem:
   - name: optaplanner
     regexs:
     - opta.*
+  - name: kie-benchmarks
```

Comment:
do you want to create the release branch for kie-benchmarks each time we prepare a release branch ?
if not, please add the `ignore_release` key
example: https://github.com/kiegroup/kogito-pipelines/blob/main/dsl/config/main.yaml#L11